startup.sh

```diff
@@ -79,6 +83,8 @@ npm install -g yo generator-rn-toolbox
 yo rn-toolbox:assets --icon ../lib/react-native/icon.png
 mv ../lib/react-native/fernet.js node_modules/fernet/fernet.js
 cd ..
-rm -rf .git lib server startup.sh
-echo "\n\n${purple}Everything is setup! You can now run `fbs run` or `react-native run-(ios|android)`${normal}"
```

Comment:
The backticks in this string actually made the shell execute the commands. This is similar to "message $(command args)". I made them single quotes.
Try this in the terminal to see what I mean:
```echo "outer `echo inner`"```

Use `$(...)` if you need to run a command in a situation like this.
> https://stackoverflow.com/questions/4708549/what-is-the-difference-between-command-and-command-in-shell-programming
> http://mywiki.wooledge.org/BashFAQ/082